build.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+// If we are going to support Windows, we need some way to cobble together
+// the build command that is portable. In other words, we can't rely on a
+// Bash subshell, `$(node...)`, to get the current Node version to provide
+// to `--target`. So we'll use the scripting language at our disposal that is
+// cross platform 😃
+
+const fs = require('fs/promises')
+const path = require('path')
+const { spawn } = require('child_process')
+
+main()
```

Comment:
This lacks tests. I know it's not a lot of code but feels like we'd want coverage right?

Reply:
What are we wanting to test? That `spawn` works?